src/main/scala/io/citrine/lolo/bags/Bagger.scala

```diff
@@ -101,19 +102,19 @@ case class Bagger(
         .next()
     }
 
-    /* Learn the actual models in parallel */
-    val parIterator = new ParRange(0 until actualBags)
-    parIterator.tasksupport = new ExecutionContextTaskSupport(InterruptibleExecutionContext())
-    val (models, importances: ParSeq[Option[Vector[Double]]]) = parIterator.map { i =>
-      // get weights
-      val sampleWeights = Nib(i).zip(weightsActual).map(p => p._1.toDouble * p._2)
-
-      // Train the model
-      val meta = method.train(trainingData.toVector, Some(sampleWeights))
-
-      // Extract the model and feature importance from the TrainingResult
-      (meta.getModel(), meta.getFeatureImportance())
-    }.unzip
+    // Learn the actual models in parallel
+    val iterator = Nib.indices.toVector
```

Comment:
This variable is not an iterator so the naming seems off. But also, I don't think we need to predefine it here. Could we just do `rng.zip(Nib.indices)` ?

Reply:
We can't do `rng.zip(Nib.indices)` because `Nib.indices` is a `Range` and doesn't have an implicit `buildFrom` method. Which seemed strange to me because I think that said method has an obvious implementation. But it still seemed like a bunch of boilerplate and it would be easier to construct the iterable directly.

The reason I put it on a separate line is because I think it reads more easily to say "here's the thing we're iterating over and now we're zipping it with `rng`" rather than "we're zipping an `rng` with _this blob you now have to think about_." But I agree that the name `iterator` is incorrect.